DTML.EduBot/Controllers/MessagesController.cs

```diff
@@ -28,6 +58,46 @@ public class MessagesController : ApiController
             return response;
         }
 
+        /// <summary>
+        /// Translate text to english and send forward
+        /// </summary>
+        /// <param name="inputText"></param>
+        /// <param name="accessToken">access token for translator API</param>
+        /// <returns></returns>
+        private async Task<string> TranslateText(string inputText, string accessToken)
```

Comment:
Also, just a nit pick, but methods that are async awaitable, should ideally be suffixed with Async. i.e. "TranslateText**Async**"